Source/Examples/Avalonia/ExampleBrowser/MainWindow.axaml

```diff
@@ -42,6 +48,18 @@
           </ItemsControl.DataTemplates>
         </ItemsControl>
       </ScrollViewer>
-      <oxy:PlotView Model="{Binding Model}" Grid.Row="0" Grid.Column="1" />
+      <Grid Grid.Column="1">
+        <Grid.RowDefinitions>
+          <RowDefinition Height="*" />
+          <RowDefinition Height="Auto" />
+        </Grid.RowDefinitions>
```

Comment:
Can do `RowDefinitions="*,auto"` in Avalonia

Reply:
Thanks for the tip!